images/scripts/arch.setup

```diff
@@ -46,6 +46,7 @@ strace \
 ltrace \
 lvm2 \
 cryptsetup \
+crun \
```

Comment:
I'm interested in that -- the commit message could give a rationale. c-podman doesn't interact with crun directly, so this smells like a missing dependency of `podman`?

Reply:
Yeah, feels like we shouldn't second-guess pacman's dependency resolution here, and rather install it as users would?

Reply:
Agreed